services/tracking.ts

```diff
@@ -63,10 +62,6 @@ export async function createManualStage(
     stage.id,
     calculateDistance(startingCoordinates, endCoordinates),
   );
-
-  let speed = getStageAvgSpeedInKmh(stage);
-
-  await setStageAvgSpeed(stage.id, speed);
```

Comment:
Warum schreibst du das nicht mehr in die Datenbank? Die StageCard greift auf genau dieses Feld zu.

Zwei mögliche Varianten:

A) stage.avgSpeed in DB speichern
- die manuelle Etappe speichert es bei der Erstellung in die DB
- die automatische Etappe speichert es bei ihrer Beendigung in die DB
- StageCard greift bei beendeten Etappen auf das DB-Feld und bei aktiven Etappen auf die Statistik-Funktion zurück

B) stage.avgSpeed aus der DB schmeißen
- Das Feld wird aus dem Schema entfernt
- es wird immer die Statistik-Funktion aufgerufen -> das müsstest du dann auch in StageCard.tsx anpassen